<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Private</class>
 <widget class="QDialog" name="Private">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>486</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Rozmowa</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Opóźnienie</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="delay"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableView" name="conversationView"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLineEdit" name="msgText"/>
       </item>
       <item>
        <widget class="QPushButton" name="sendButton">
         <property name="text">
          <string>Wyślij</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
